File: min/Min.java
public class Min {
    public static void main(String[] args) {
        int[] myList = {3, 1, 2};
        int min = getMin(myList);
        System.out.println(min);
    }
    
    public static int getMin(int[] inputArray){ 
        int minValue = inputArray[0]; 
        for(int i=1;i<inputArray.length;i++){ 
            if(inputArray[i] < minValue){ 
                minValue = inputArray[i]; 
            } 
        } 
        return minValue; 
    }
}

File: min_in_array/min.c
#include <stdio.h>

static int minValue(const int* array) {
    size_t i;
    int min;
    min = array[0];
    for (i = 1; i < sizeof(array)/sizeof(array[0]); ++i) {
        if (min > array[i]) {
            min = array[i];
        }
    }
    return min;
}

int main(void) {
    const int array[] = {3, 1, 2};
    printf("%i\n", minValue(array) );
}